client/me/purchases/manage-purchase/index.jsx

```diff
@@ -142,13 +153,17 @@ const ManagePurchase = React.createClass( {
 		upgradesActions.addItems( renewItems );
 
 		page( '/checkout/' + this.props.selectedSite.slug );
-	},
+	}
 
 	renderRenewButton() {
 		const purchase = getPurchase( this.props );
 		const { translate } = this.props;
 
-		if ( ! config.isEnabled( 'upgrades/checkout' ) || ! isRenewable( purchase ) || isExpired( purchase ) || isExpiring( purchase ) || ! getSelectedSite( this.props ) ) {
+		if ( ! config.isEnabled( 'upgrades/checkout' ) ) {
+			return null;
+		}
+
+		if ( ! isRenewable( purchase ) || isExpired( purchase ) || isExpiring( purchase ) || ! getSelectedSite( this.props ) ) {
```

Comment:
Sure, but this section already uses these utilities, and I didn't want to add to this PR by removing them - like you said, out of scope for this PR.